eng/Signing.props

```diff
@@ -1,3 +1,123 @@
+      here and rely on overall signing validation.
+    -->
+    <AllowEmptySignList>true</AllowEmptySignList>
+  </PropertyGroup>
+
+  <!-- Get artifact locations to sign. -->
+  <Import Project="$(RepositoryEngineeringDir)/liveBuilds.targets" />
+
+  <ItemGroup>
+    <!--
+      Replace the default items to sign with the specific set we want. This allows the build to call
+      Arcade's Sign.proj multiple times for different sets of files as the build progresses.
+    -->
+    <ItemsToSign Remove="@(ItemsToSign)" />
+
+    <!-- Find bundle artifacts, which need multiple stages to fully sign. -->
+    <BundleInstallerEngineArtifact Include="$(ArtifactsPackagesDir)**/*engine.exe" />
+    <BundleInstallerExeArtifact Include="$(ArtifactsPackagesDir)**/*.exe" />
+
+    <!-- apphost and comhost template files are not signed, by design. -->
+    <FileSignInfo Include="apphost.exe;comhost.dll" CertificateName="None" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(CrossTargetComponentFolder)' != ''">
+    <CoreCLRCrossTargetItemsToSign Include="$(CoreCLRArtifactsPath)$(CrossTargetComponentFolder)/sharedFramework/*.dll" />
+    <CoreCLRCrossTargetItemsToSign Include="$(CoreCLRArtifactsPath)$(CrossTargetComponentFolder)/sharedFramework/*.exe" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(SignBinaries)' == 'true'">
+    <!-- Sign CoreCLR. -->
+    <ItemsToSign Include="$(CoreCLRSharedFrameworkDir)*.dll" />
+    <ItemsToSign Include="$(CoreCLRSharedFrameworkDir)*.exe" />
+
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)System.Private.CoreLib.dll" />
+
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/crossgen2.exe" />
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/crossgen2.dll" />
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/ILCompiler.DependencyAnalysisFramework.dll" />
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/ILCompiler.ReadyToRun.dll" />
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/ILCompiler.TypeSystem.ReadyToRun.dll" />
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/clrjitilc.dll" />
+    <ItemsToSign Include="$(CoreCLRArtifactsPath)crossgen2/jitinterface.dll" />
+
+    <ItemsToSign Include="@(CoreCLRCrossTargetItemsToSign)" />
+
+    <FileSignInfo Include="mscordaccore.dll" CertificateName="MicrosoftSHA2" />
+
+    <!-- Sign api-ms-win-core-xstate-l2-1-0 binary as it is only catalog signed in the current SDK. -->
+    <ItemsToSign
+      Condition="'$(ConfigurationGroup)' == 'Release' and '$(TargetArchitecture)' == 'x86'"
+      Include="$(CoreCLRArtifactsPath)Redist\ucrt\DLLs\$(TargetArchitecture)\api-ms-win-core-xstate-l2-1-0.dll" />
+
+    <!-- Sign libraries. -->
+    <ItemsToSign Include="$(LibrariesNativeArtifactsPath)*.dll" />
+    <ItemsToSign Include="$(LibrariesSharedFrameworkRefArtifactsPath)*.dll" />
+    <!-- Most runtime artifacts will be crossgenned, so sign them post-crossgen. mscorlib isn't. -->
```

Comment:
I guess the crossgen'd libraries artifacts are signed as part of the nupkg signing?

Reply:
No, they're signed in `SignR2RBinaries`. They have to be signed before getting put in the MSIs because Arcade can't recursively sign them (or the bundle installer EXEs).

Reply:
Ok, cool. The way we need to sign is confusing now, I was used to the signing we had, that we just signed nupkgs 😄

Reply:
> Ok, cool. The way we need to sign is confusing now

Very. Welcome to the life of core-setup. 😉 At least I didn't need to bring the confusing publishing over.